java/src/main/native/CMakeLists.txt

```diff
@@ -240,3 +240,18 @@ else()
   # Tell CMake what CUDA language runtime to use
   set_target_properties(cudfjni PROPERTIES CUDA_RUNTIME_LIBRARY Shared)
 endif()
+
+# ##################################################################################################
+# * install shared libraries ----------------------------------------------------------------------
+if(TARGET nvcomp::nvcomp)
+  get_target_property(NVCOMP_LOCATION nvcomp::nvcomp LOCATION)
+  get_filename_component(NVCOMP_LIB_PATH "${NVCOMP_LOCATION}" DIRECTORY)
+
+  add_custom_command(
+    TARGET cudfjni PRE_LINK COMMAND ${CMAKE_COMMAND} -E copy
```

Comment:
rather than explicitly grabbing the parent directory and assuming gdeflate and bitcomp are there as well, I think it would be simpler and safer if we could pass the nvcomp targets to the custom command directly. IIRC cmake will automatically expand target names to the file path of the target when passed to a custom command.  If so, it would be great if we could pass nvcomp::nvcomp, nvcomp::bitcomp, and nvcomp::gdeflate (or whatever nvcomp calls its sub-library targets).